packages/office-addin-project/README.md

```diff
@@ -20,6 +20,7 @@ Syntax:
 
 `manifest-path`: path to manifest file.
 `backup-path`: path to the generated backup of the project.
+`output-directory`: path to the output folder for the converted manifest.
```

Comment:
That is just an option, there is a default directory `./manifest` if nothing is chosen. Do you think it is better to not even have the option?

Reply:
I don't think we want any new option as it can get confusing and make us responsible for making sure everythign works with the new manifest location (like our npm scripts).  We should have a standard way of updating the project and if customer want to re-organize their project after the fact they can do it.